features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumApplicationScope.kt

```diff
@@ -77,6 +78,7 @@ internal class RumApplicationScope(
                 syntheticsTestId = event.testId,
                 syntheticsResultId = event.resultId
             )
+            Log.i(RumScope.SYNTHETICS_LOGCAT_TAG, "_dd.application.id=${rumContext.applicationId}")
```

Comment:
Is this safe to expose it in the Logcat ? The session Id and view Id is something we generated but the `applicationId` is the actual RUM application ID.

Reply:
It's only exposed when being ran via Synthetics. 
Also the RUM Application ID is an identifier, not a token or a key. Just having the App id gives no permission by itself